app/views/expenses/show.html.erb

```diff
@@ -0,0 +1,46 @@
+<p style="color: green"><%= notice %></p>
+<%= link_to categories_path do %>
+  <%= image_tag 'back-icon.png', alt: 'Back', class: 'back-icon' %>
+<% end %>
+
+<div class="container-fluid new">
+  <div class="row justify-content-center">
+    <div class="col-8">
+      <div class="row head">
+        <h2><%= @category.name %></h2>
+        <p><%= @category.created_at.strftime('%e%p %B %Y').strip %></p>
+      </div>
+
+      <div class="row shw">
+        <% if @category.expenses.present? %>
+          <% @expenses.each do |expense| %>
+            <div class="row">
+              <div class="col">
+                <h5><%= expense.name %> - <%= number_to_currency(expense.amount, unit: '$') %></h5>
+              </div>
+              <div class="col">
+                <%= link_to expense, method: :delete, class:'btn',data: { confirm: 'Are you sure?' } do %>
+                  <%= image_tag 'blue-delete.png', alt: 'Delete', class: 'delete-icon' %>
+                <% end %>
+              </div>
+              <div class="col">
+                <%= link_to edit_expense_path(expense) do %>
+                  <%= image_tag 'edit-icon.png', alt: 'Edit', class: 'edit-icon' %>
+                <% end %>
+              </div>
+            </div>
+          <% end %>
+        <% else %>
+          <p>No expenses for this category.</p>
+        <% end %>
+      </div>
+    </div>
+  </div>
```

Comment:
- [ ] Kindly improve the design of the expenses cards to match the design.

  Your Implementation | The Design
  :---:|:---:
  ![image](https://github.com/Tracy-miranja/budget/assets/36542986/ec463f31-5dd8-43be-ad45-f1562eefccc6) | ![image](https://github.com/Tracy-miranja/budget/assets/36542986/49d7e74a-c333-4068-9695-73719191abe4)